### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by aminbksi**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<!--
+
+Submitting a Pull Request
+
+- Choose a meaningful title for this pull request.
+- Please fill out all sections of this form. You can delete the helpful comments.
```
Comment (maasva) [inline .github/pull_request_template.md:6]: Why would you delete the helpful comments?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- Pull Requests without clear information will take longer and may even be rejected.
+
+-->
+
+### Description
+
+<!--
+
+Clearly describe the submitted changes in lots of detail. Include images where helpful. Be as thorough as possible. You can use MarkDown syntax to improve readability with bullet lists, code blocks, and so on. PREVIEW and fix up formatting before submitting.
```
Comment (maasva) [inline .github/pull_request_template.md:15]: remove "lots of"

Comment (maasva) [reply .github/pull_request_template.md:15]: why is PREVIEW in all caps

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+<!-- Does this PR require a specific requirement? -->
+
+### Configurations
+
+<!-- If applicable attach Configurations ZIP and any other files needed to test this PR. -->
+
+### Related Issues
+
+<!-- Does this PR fix a bug or fulfill a Feature Request? Link the related Issues or Feature request here. -->
```
Comment (maasva) [inline .github/pull_request_template.md:29]: If it is not part of an issue, it shouldn't have been made. If not, they would have to explain why they're even making the request.

**Review by maasva — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (aminbksi) [reply .github/pull_request_template.md:6]: placeholder text

File: .github/pull_request_template.md (hunk 2 above)
Comment (aminbksi) [reply .github/pull_request_template.md:15]: emphasising

**Pull request merged**